packages/phone-number/lib/parse/index.test.js

```diff
@@ -19,34 +19,15 @@ describe('parse', () => {
     expect(parse({ input: '_#' }).isValid).toBe(false);
   });
 
-  test('parse correct number', () => {
```

Comment:
Why are you loosening the test cases and not making the tests more comprehensive? The original tests were written during an initial refactor to ensure the functionality remains the same. What is the reason for making the tests even more loose?